microsoft-edge/webview2/how-to/print.md

```diff
@@ -798,23 +796,23 @@ async void PrintToPdfStream()
   {
     string title = webView.CoreWebView2.DocumentTitle;
 
-    // Passing null for `PrintSettings` results in default print settings used.
+    // Passing null for PrintSettings causes the default print settings to be used.
     System.IO.Stream stream = await webView.CoreWebView2.PrintToPdfStreamAsync(null);
     DisplayPdfDataInPrintDialog(stream);
 
     MessageBox.Show(this, "Printing " + title + " document to PDF Stream " +
-                ((stream != null) ? "succedded" : "failed"), "Print To PDF Stream");
+                ((stream != null) ? "succeeded" : "failed"), "Print To PDF Stream");
```

Comment:
@monica-ch - tech review this change in code, a typo fix from succedded to succeeded

Reply:
the fix in displayed string is ok